lib/_stream_writable.js

```diff
@@ -175,8 +175,7 @@ function writeAfterEnd(stream, cb) {
 // watermarks determine how many objects to keep in the buffer, rather than
 // how many bytes or characters.
 function validChunk(stream, state, chunk, cb) {
-  var valid = true;
-  var er = false;
+  var er;
```

Comment:
Crankshaft doesn't deal well with `let` inside loops. In this case I don't expect a difference.